app/src/main/java/com/infomaniak/mail/data/api/ApiRoutes.kt

```diff
@@ -127,11 +131,11 @@ object ApiRoutes {
     }
 
     fun folders(mailboxUuid: String): String {
-        return "${mailMailbox(mailboxUuid)}/folder"
+        return "${mailMailbox(mailboxUuid)}/folder?with=ik-static"
     }
 
     private fun folder(mailboxUuid: String, folderId: String): String {
-        return "${folders(mailboxUuid)}/$folderId"
+        return "${mailMailbox(mailboxUuid)}/folder/$folderId"
```

Comment:
Did it not work previously?

Reply:
The `folders()` call was updated to handle the `ik-static` folders.
But because of that, it's not usable anymore by another calls, so this `folder()` call needed to be updated also.